packages/plugins/graphql/server/bootstrap.js

```diff
@@ -61,6 +61,9 @@ module.exports = async ({ strapi }) => {
         ? ApolloServerPluginLandingPageDisabled()
         : ApolloServerPluginLandingPageGraphQLPlayground(),
     ],
+
+    // Required from version 3.9 https://www.apollographql.com/docs/apollo-server/performance/cache-backends/
```

Comment:
I think we can remove the comment, no need for it.

Reply:
Might also be better to set this as a configuration option instead too.

Reply:
I'll remove the comment

It could be changed as user wish, as mentioned https://github.com/strapi/strapi/issues/13646#issuecomment-1185811600 without issues